tests/test_bulk_tx_multiep/src/main.xc

```diff
@@ -1,47 +1,46 @@
 // Copyright 2016-2021 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
-#include <xs1.h>
-#include <print.h>
-#include <stdio.h>
-#include "xud.h"
-#include "platform.h"
-#include "shared.h"
-
-#define XUD_EP_COUNT_OUT   5
-#define XUD_EP_COUNT_IN    8
 
-/* Endpoint type tables */
-XUD_EpType epTypeTableOut[XUD_EP_COUNT_OUT] = {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL};
-XUD_EpType epTypeTableIn[XUD_EP_COUNT_IN] =   {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, 
-                                               XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL};
+#define EP_COUNT_OUT       (4)
+#define EP_COUNT_IN        (8)
 
-#define PKT_LENGTH_START 10
-#define PKT_LENGTH_END 19
+#define PKT_LENGTH_START   (10)
+#define PKT_LENGTH_END     (19)
 
+#include "shared.h"
 
-int main()
+XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
+                                                XUD_EPTYPE_BUL,
```

Comment:
It appears that this line has a different level of indentation that the ones which follow it.  Is that intentional?

Reply:
Fixed